src/view/screens/PreferencesFollowingFeed.tsx

```diff
@@ -92,26 +92,28 @@ export function PreferencesFollowingFeed({navigation}: Props) {
   )
 
   return (
-    <CenteredView
-      testID="preferencesHomeFeedScreen"
-      style={[
-        pal.view,
-        pal.border,
-        styles.container,
-        isTabletOrDesktop && styles.desktopContainer,
-      ]}>
-      <ViewHeader title={_(msg`Following Feed Preferences`)} showOnDesktop />
-      <View
+    <View testID="preferencesHomeFeedScreen" style={s.hContentRegion}>
+      <SimpleViewHeader
+        showBackButton={isMobile}
         style={[
-          styles.titleSection,
-          isTabletOrDesktop && {paddingTop: 20, paddingBottom: 20},
+          pal.border,
+          {borderBottomWidth: 1},
+          !isMobile && {borderLeftWidth: 1, borderRightWidth: 1},
         ]}>
-        <Text type="xl" style={[pal.textLight, styles.description]}>
-          <Trans>Fine-tune the content you see on your Following feed.</Trans>
-        </Text>
-      </View>
+        <View style={{flex: 1}}>
+          <Text type="title-lg" style={[pal.text, {fontWeight: 'bold'}]}>
+            <Trans>Following Feed Preferences</Trans>
+          </Text>
+          <Text style={pal.textLight}>
+            <Trans>Fine-tune the content you see on your Following feed.</Trans>
+          </Text>
+        </View>
+      </SimpleViewHeader>
```

Comment:
Can we move this into the scroll view? Smaller screens w/ font scaling might have trouble viewing the settings.

![Screenshot 2024-07-24 at 12 15 40 PM](https://github.com/user-attachments/assets/6782c5de-9124-46fe-a099-2ce4f3167df4)